iceoryx_posh/test/moduletests/test_mepoo_shared_chunk.cpp

```diff
@@ -261,3 +261,22 @@ TEST_F(SharedChunk_Test, MultipleChunksCleanup)
     EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
     EXPECT_THAT(chunkMgmtPool.getUsedChunks(), Eq(1));
 }
+
+TEST_F(SharedChunk_Test, NotEqualWithAnotherMemoryChunk)
```

Comment:
I would suggest to add those test into the `Compare**` test part of the unit test.
Additionally, I would rename this test into `DefaultConstructedChunkIsNotEqualToValidChunk`